dora/shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -170,23 +170,22 @@ public final class LsCommand extends AbstractFileSystemCommand {
    * @param groupName group name
    * @param size size of the file in bytes
    * @param timestamp the epoch time in ms
-   * @param inAlluxioPercentage whether the file is in Alluxio
    * @param persistenceState the persistence state of the file
    * @param path path of the file or folder
    * @param dateFormatPattern the format to follow when printing dates
    * @return the formatted string according to acl and isFolder
    */
   public static String formatLsString(boolean hSize, boolean acl, boolean isFolder, String
       permission,
-      String userName, String groupName, long size, long timestamp, int inAlluxioPercentage,
+      String userName, String groupName, long size, long timestamp,
       String persistenceState, String path, String dateFormatPattern) {
     String inAlluxioState;
     String sizeStr;
     if (isFolder) {
       inAlluxioState = IN_ALLUXIO_STATE_DIR;
       sizeStr = String.valueOf(size);
     } else {
-      inAlluxioState = String.format(IN_ALLUXIO_STATE_FILE_FORMAT, inAlluxioPercentage);
```

Comment:
we can probably just remove this column

Reply:
Thanks @jja725 for the suggestion. Instead of removing it, I change it to a fixed string FILE which keeps consistent with DIR format, and it also clearly tell users it is a file. WDYT? thanks.